src/Console/BaseCommand.php

```diff
@@ -45,7 +45,7 @@ abstract class BaseCommand implements CommandInterface
      */
     public function setName(string $name)
     {
-        if (strpos($name, ' ') < 1) {
+        if (!strpos($name, ' ')) {
```

Comment:
That's an odd check. Might be cleaner to trim it first.

Reply:
Can't just trim it off, there's a specific test which throws an exception for leading space.